Source/WPEFramework/PluginServer.cpp

```diff
@@ -125,10 +125,14 @@ namespace PluginHost
             return (result);
         }
         //! Allow a JSONRPC message to be checked before it is offered for processing.
-        virtual bool Allowed(const Core::JSONRPC::Message& message) const override
+        bool Allowed(const Core::JSONRPC::Message& message) const override
         {
             return ((_hasSecurity == false) || CheckMessage(message));
         }
+        string Token(void) const override
```

Comment:
I would have expected string Token(), so without void

Reply:
Me too, seen that I am quite lazy, I can not imagen that I typed it so propbably I copy pasted it. However is does not hurt any functionality. Maybe it is this automatic style tool that did this. Interesting...